controllers/api.controller.js

```diff
@@ -0,0 +1,5 @@
+const endpoints = require("../endpoints.json");
+
+exports.getApi = (req, res) => {
+  res.status(200).send(endpoints);
```

Comment:
better to send an object with a property of endpoints instead of only the endpoints object